lab1.js

```diff
@@ -109,6 +132,11 @@ var nextAnimal;
 
 // your code goes here
 
+var nextAnimal = 'Lion';
+var nextAnimal = Math.random();
```

Comment:
here, you're just assigning a random number to nextAnimal. The question wants you to use math.random as part of a line of code that selects a random animal from the favoriteAnimals array.